dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/LKSGData.kt

```diff
@@ -0,0 +1,221 @@
+package org.dataland.datalandbackend.model.lksg
+
+import org.dataland.datalandbackend.annotations.DataType
+import org.dataland.datalandbackend.model.enums.commons.YesNo
+import java.math.BigDecimal
+import java.time.LocalDate
+
+/**
+ * --- API model ---
+ * Fields of the questionnaire for the LKSG framework
+ */
+@DataType("lksg")
+data class LKSGData(
+    val dataDate: LocalDate? = null,
+
+    val LKSGinScope: YesNo? = null,
```

Comment:
When running detekt does this work? I think our detekt rules forbid the use of capital letters

Reply:
This file no longer exists